src/dom/dom.js

```diff
@@ -2212,7 +2212,7 @@ if (navigator.mediaDevices.getUserMedia === undefined) {
 p5.prototype.createCapture = function(...args) {
```

Comment:
Can we update the doc comments above this to mention the `flipped` parameter, and maybe add an example that uses it?

Reply:
I will soon update the inline documentation and give a simple good-to-go example.